### PR TITLE
add fis token

### PR DESCRIPTION
**Pull request by signalxu**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by signalxu** [comment]
> recheck

recheck

**Comment by signalxu** [comment]
recheck

**Comment by signalxu** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by signalxu** [comment]
recheck

**Comment by signalxu** [comment]
https://api.coinpaprika.com/v1/coins/fis-stafi

**Comment by signalxu** [comment]
@jeff-dude  Hey, could you please advise me on how to address the failed checks?


https://api.coinpaprika.com/v1/coins/fis-stafi

File: models/prices/ethereum/prices_ethereum_tokens.sql
Comment (jeff-dude) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1]: you don't need to alter this file at this time, as it hasn't been migrated yet (open PR currently)

Comment (signalxu) [reply models/prices/ethereum/prices_ethereum_tokens.sql:1]: fixed

**Comment by signalxu** [comment]
@jeff-dude  anything else I need to do about this pr?

**Comment by jeff-dude** [comment]
> @jeff-dude anything else I need to do about this pr?

since my last comment, we have merged the dunesql prices spells. i went ahead and changed for ya to avoid the hassle.

**Pull request merged**